examples/react-app/src/Forms/SlidersForm.tsx

```diff
@@ -0,0 +1,143 @@
+import { useState } from 'react';
+import { PlmgButton, PlmgSlider } from '@ducky/plumage-react';
+
+export default function SlidersForm() {
+  const SLIDERS = {
+    stepped: {
+      label: 'stepped',
+      default: 25,
+    },
+    decimal: {
+      label: 'decimal',
+      default: 0.5,
+    },
+    large: {
+      label: 'large',
+      default: 50,
+    },
+  };
+
+  const [sliders, setSliders] = useState({
+    stepped: SLIDERS.stepped.default,
+    decimal: SLIDERS.decimal.default,
+    large: SLIDERS.large.default,
+  });
+
+  const handleChange = (value: number, target: HTMLElement) => {
+    const elementName = String(target.getAttribute('name'));
+    if (elementName) {
+      setSliders({ ...sliders, [elementName]: value });
+    }
+  };
+
+  const handleReset = (target: HTMLElement) => {
+    const elementLabel = target.getAttribute('aria-label');
+    if (elementLabel) {
+      setSliders({
+        ...sliders,
+        [elementLabel]: SLIDERS[elementLabel].default,
+      });
+    }
+  };
```

Comment:
You have added two handler functions here which are calling the same thing; `setSlider`, which means they need the same information. I suggest simplifying and passing in your variable rather than cleverly deduce it from the element. How this form is implemented allows you to do that.
Remove one of the handers and pass in the slider identifier to be changed in state.  

Eg
```  
const handleChange = (value: number, slider: string) => {
    setSliders({ ...sliders, [slider]: value });
 };
 ```
Used via:
```
onValueUpdated={(event) =>
    handleChange(event.detail.value, SLIDERS.decimal.label)
}
onClick={() =>
    handleChange(SLIDERS.decimal.default, SLIDERS.decimal.label)
}
```

Reply:
In fact, why create a handler at all and just call `setSliders` directly.

eg:
```
onValueUpdated={(event) => {
  setSliders({
    ...sliders,
    [SLIDERS.stepped.label]: event.detail.value,
  });
}}
onClick={() => {
  setSliders({
    ...sliders,
    [SLIDERS.stepped.label]: SLIDERS.stepped.default,
  });
}}
```

Reply:
Yes, this is obviously the simplest solution! I had thought it was bad practice to repeat the same function when it could be abstracted out, but abstracting it out just made things overly complicated.

I had thought about creating an array of components and then mapping over them but I decided that this would be less readable.